geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -291,12 +291,15 @@ public static long zinterstore(RegionProvider regionProvider, RedisKey key,
       List<ZKeyWeight> keyWeights,
       ZAggregator aggregator) {
     List<RedisSortedSet> sets = new ArrayList<>(keyWeights.size());
+    MemberMap interMembers = new MemberMap(keyWeights.size());
+    ScoreSet interScores = new ScoreSet();
```

Comment:
Rather than moving the definition of these variables up to here, they can stay where they were originally on lines 312 and 313 and instead just directly use `new ScoreSet()` and `new MemberMap(0)` in the `sortedSetOpStoreResult()` method call. The initial size of the `MemberMap` in `zinterstore()` should remain as `smallestSet.getSortedSetSize()` as that's the largest size it's possible for the result to be.